widgetssdk/src/main/java/com/glia/widgets/chat/data/GliaChatRepository.java

```diff
@@ -71,6 +71,21 @@ public void sendMessageAttachment(MessageAttachment attachment, Listener listene
         );
     }
 
+    public void sendResponse(String cardMessageId, String text, String value, Listener listener) {
+        Dependencies.glia().getCurrentEngagement().ifPresent(engagement ->
+                engagement.getChat().sendResponse(cardMessageId, text, value, (result, ex) -> {
```

Comment:
I would suggest renaming `ex` to `exception` or `error`. It's more commonly used I suppose.

Reply:
Agree with you. But other similar parameters in this class are named as `ex`.